lib/top_hangman/file_manager.rb

```diff
@@ -0,0 +1,104 @@
+require 'yaml'
+
+module TopHangman
+  class FileManager
+    ROOT = 'bin/games/'.freeze
+    SAVED = :saved
+
+    def initialize(game)
+      @saved_games = nil
+      @selected_game = nil
+      @current_game = game
+    end
+
+    def list_all_saved_games
+      @saved_games = Dir["#{ROOT}*"].map { |file| File.basename(file) }
+    end
+
+    def select_game(number)
+      @selected_game = @saved_games[number - 1]
+    end
+
+    def save_game(name_tu_use = nil)
+      current_game_hash = build_game_for_file(@current_game)
+      file_name = name_tu_use || build_file_name
+      file = File.open("#{ROOT}#{file_name}", 'w')
+      file.write(current_game_hash.to_yaml)
+      file.close
+
+      SAVED
+    end
+
+    def load_game(file_to_load)
+      file = File.open("#{ROOT}#{file_to_load}", 'r')
+      game_from_file = YAML.load(file.read)
```

Comment:
Just a small suggestion, I think you can use `YAML.load_file('some.yml')` instead of these two lines

Reply:
noooo :(

I was following [this](https://medium.com/@kristenfletcherwilde/saving-retrieving-data-with-a-yaml-file-in-ruby-the-basics-e45232903d94)